keps/sig-cli/1441-kubectl-debug/README.md

```diff
@@ -651,8 +651,8 @@ integration test for [Pod Troubleshooting with Ephemeral Debug Container
 
 #### Beta -> GA Graduation
 
-- [ ] Test plan GA milestones reached
-- [ ] User feedback gathered over 2 release cycles.
+- [x] Test plan GA milestones reached
+- [x] User feedback gathered over 2 release cycles.
 - [ ] 3 external articles suggest using `kubectl debug`
```

Comment:
Did a quick google search and I got (in no particular order):
- https://collabnix.com/debug-your-kubernetes-applications-made-easy-with-kubectl-debug-tool/
- https://www.linkedin.com/pulse/debug-kubernetes-applications-kubectl-cuong-nguyen-duc
- https://medium.com/datamindedbe/debugging-running-pods-on-kubernetes-2ba160c47ef5

Reply:
Thanks a lot. Updated with the links above.